src/components/Pagination/Pagination.tsx

```diff
@@ -87,17 +87,17 @@ export const Pagination = ({
   const navClasses = classnames('usa-pagination', className)
 
   const isOnFirstPage = currentPage === 1
-  const isOnLastPage = currentPage === totalPages
+  const isOnLastPage = totalPages ? currentPage === totalPages : null
 
-  const showOverflow = totalPages > maxSlots // If more pages than slots, use overflow indicator(s)
+  const showOverflow = totalPages ? totalPages > maxSlots : true // If more pages than slots, use overflow indicator(s)
 
   const middleSlot = Math.round(maxSlots / 2) // 4 if maxSlots is 7
   const showPrevOverflow = showOverflow && currentPage > middleSlot
   const showNextOverflow =
-    showOverflow && totalPages - currentPage >= middleSlot
+    totalPages ? showOverflow && totalPages - currentPage >= middleSlot : true
```

Comment:
Though: This feels a bit hard to read for a ternary. I wonder if we should write this as an if statement or a small function.